src/main/java/util/DateConstants.java

```diff
@@ -0,0 +1,16 @@
+package util;
+
+import java.time.LocalDate;
+import java.time.ZoneId;
+
+public class DateConstants {
```

Comment:
It depends on how we want to structure the superclasses but would it make sense to have these constants within the `Import`/`Export` classes? Might just be subjective, as both ways work.